packages/backend/src/api/controllers/UserController.ts

```diff
@@ -123,7 +127,10 @@ export class UserController {
       tradingMode: this.tradingMode,
       starkKey,
       ethereumAddress: registeredUser?.ethAddress ?? EthereumAddress.ZERO,
-      withdrawableAssets: [],
+      withdrawableAssets: withdrawableAssets.map((asset) => ({
+        asset: { hashOrId: asset.assetHash },
```

Comment:
Don't we need full asset details here to properly display its name and amount? I think it's necessary to reach to `assetDetailsMap` first to get it.